src/include/distributed/citus_clauses.h

```diff
@@ -15,9 +15,25 @@
 #include "nodes/nodes.h"
 #include "nodes/parsenodes.h"
 
+
+/*
+ * This struct is used to pass information to master
+ * evaluation logic.
+ */
+typedef struct MasterEvaluationContext
+{
+	PlanState *planState;
+	bool evaluateParams;
+	bool evaluateFunctions;
```

Comment:
maybe use `evaluateExpressions` instead of `evaluateFunctions `?

Reply:
I think evaluateFunctions is fine and consistent with the rest of the code.